content/developers/api-reference/members-api/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Members API"
+description: "Members API Reference"
+lead: "Members API Reference"
+date: 2021-06-09T13:29:57+01:00
+lastmod: 2021-06-09T13:29:57+01:00
+draft: false
+images: []
+menu: 
+  developers:
+    parent: "api-reference"
+weight: 114
+toc: true
+aliases: 
+  - /docs/api-reference/members-api/
+---
+{{< note >}}
+**Note:** This page is primarily intended for developers who will be writing applications that will use DataTrails for provenance. 
+If you are looking for a simple way to test our API you might prefer our [Postman collection](https://www.postman.com/datatrails-inc/workspace/datatrails-public/overview), the [YAML runner](/developers/yaml-reference/story-runner-components/) or the [Developers](https://app.datatrails.ai) section of the web UI. 
+
+Additional YAML examples can be found in the articles in the [Overview](/platform/overview/introduction/) section.
+{{< /note >}}
+
+## Members API Examples
+
+Create the [bearer_token](/developers/developer-patterns/getting-access-tokens-using-app-registrations) and store in a file in a secure local directory with 0600 permissions.
+
+### Listing members of your tenant
+
+To fetch the list of members, simply `GET` the `/members` resource
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     https://app.datatrails.ai/archivist/iam/v1/members
+```
+
+By default this doesn't show members that are deactivated. You can filter the list to include 
+deactivated users:
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     https://app.datatrails.ai/archivist/iam/v1/members?member_state=MEMBER_STATE_BOTH
+```
+
+### Promoting a member of your tenant
+
+You can manage the roles of a user (i.e. promoting or demoting them by way of the 'OWNER' role) by
```

Comment:
```suggestion
You can manage the roles of a member (i.e. promoting or demoting them by way of the 'OWNER' role) by
```

Reply:
its a bit jarring how the UI refers to these as users, but the API as members? maybe we want to be consistent at least on a customer facing level? if so we can disregard this suggestion?

Reply:
What are your thoughts on this one @JAG-UK ? I know using the term 'Users' in the UI was requested. Is it worth the inconsistency overall?

Reply:
They're conceptually different: a user having roles is about what a person (or piece of software) can do within the system. The way we express and manage that is through Tenancy Membership. So a 'user' is a 'member' of a number of 'tenancies'.

I'll make a suggestion in a separate comment

Reply:
Users only have roles in the context of a particular tenant though, i.e. their membership